src/manifest.json

```diff
@@ -0,0 +1,5 @@
+{
+  "name": "Tokenizer",
+  "iconPath": "./assets/icons/icon-512x512.png",
+  "description": "Tokenizer tokenizes tokenizable assets."
+}
```

Comment:
Comment somewhere why it's here.

Add link:
https://docs.gnosis-safe.io/build/sdks/safe-apps/releasing-your-safe-app#get-your-safe-app-listed-in-the-safe